eng/CredScanSuppression.json

```diff
@@ -1,4 +1,48 @@
 {
     "tool": "Credential Scanner",
-    "suppressions": []
+    "suppressions": [
+        {
+            "placeholder": [
+                "XxxxXxxxXXXxxxXXXXxxxxXXxxxXxxx",
+                "HusH_Sec4et",
+                "myPassw0rd",
+                "myPassw0rd3",
+                "SWsSsd__233$Sdsds#%Sd!",
+                "kt#_gahr!@aGERDXA",
+                "Aa1!zyx_",
+                "Aa!1()-xyz",
+                "000000000000000000000000000000000000000000000000000",
+                "UsernameAndPassword",
+                "b4GrSv5ajm8zBPgaihzmWtzw5BCiUYjAxZ%2FqOgGQZao%3D&response_type=client_credentials",
+                "123456",
+                "Password1!",
+                "NewPassword1!",
+                "MyStr0ngP4ssword",
+                "secret_password",
+                "secret",
+                "123",
+                "connectionstring",
+                "123_abc",
+                "pa$$w0rd",
+                "Password1!!!",
+                "NzhL3hKZbJBuJ2484dPTR+xF30kYaWSSCbs2BzLgVVI1woqeST/1IgqaLm6QAOTxtGvxctSNbIR/1hW8yH+bJg==",
```

Comment:
Is this really a placeholder? Some of these look like keys.

Reply:
Right, those are symmetricKey. Will remove and work with team to use some static ones.